changelog/generated/v1.31.0.md

```diff
@@ -0,0 +1,22 @@
+## v1.31.0
+
+### Changes
+
+- (ansible): Update the `quay.io/operator-framework/ansible-operator` base image to now use Ansible 2.15.0.  Removes pre-installed Python modules that are not required to build and run the `testdata/ansible/memcached-operator` sample. Updates Python to 3.9 to work with Ansible 2.15.0. ([#6483](https://github.com/operator-framework/operator-sdk/pull/6483))
+- (ansible/v1): Updates the `ansible/v1` plugin scaffolding to: - Remove the `community.kubernetes` collection - Update the `operator_sdk.util` collection from `v0.4.0` --> `v0.5.0` - Update the `molecule.yaml` files to work with `molecule v5.1.0`. ([#6483](https://github.com/operator-framework/operator-sdk/pull/6483))
+- (helm): Use informer to list helm secrets to improve performance. ([#6354](https://github.com/operator-framework/operator-sdk/pull/6354))
+- Updated Reference from " to_be_owned.namespace" to " to_be_owned.metadata.namespace " at line 186.
```

Comment:
This is a part of helm? Maybe we missing a reference PR here, or it is a part of previous one?

Reply:
It is missing a reference PR, I just added the link to the PR so it should be updated now.